Makefile

```diff
@@ -70,88 +70,31 @@ dev-agent-image:
 	kubectl -n zarf set image deployment/agent-hook server=$(tag)
 
 init-package: ## Create the zarf init package, macos "brew install coreutils" first
-	$(ZARF_BIN) package create --confirm --architecture amd64
-	$(ZARF_BIN) package create --confirm --architecture arm64
-	mv zarf-init-*.tar.zst build
-	cd build && sha256sum -b zarf* > zarf.sha256
-	ls -lh build
+	@test -s $(ZARF_BIN) || $(MAKE) build-cli
 
-build-test: build-cli init-package ## Build the CLI and create the init package
+	@test -s ./build/zarf-init-$(ARCH).tar.zst || $(ZARF_BIN) zarf.yaml build -a $(ARCH) --confirm 
 
 ci-release: init-package ## Create the init package
 
-.PHONY: package-example-game
-package-example-game: ## Create the Doom example
-	cd examples/game && ../../$(ZARF_BIN) package create --confirm && mv zarf-package-* ../../build/
+build-examples:
+	@test -s $(ZARF_BIN) || $(MAKE) build-cli
 
-.PHONY: package-example-component-scripts
-package-example-component-scripts: ## Create component script example
-	cd examples/component-scripts && ../../$(ZARF_BIN) package create --confirm && mv zarf-package-* ../../build/
+	@test -s ./build/zarf-package-dos-agmes-$(ARCH).tar.zst || $(ZARF_BIN) examples/game build -a $(ARCH) --confirm
+	
+	@test -s ./build/zarf-package-component-scripts-$(ARCH).tar.zst || $(ZARF_BIN) examples/component-scripts build -a $(ARCH) --confirm
 
-.PHONY: package-example-component-choice
-package-example-component-choice: ## Create component choice example
-	cd examples/component-choice && ../../$(ZARF_BIN) package create --confirm && mv zarf-package-* ../../build/
+	@test -s ./build/zarf-package-component-choice-$(ARCH).tar.zst || $(ZARF_BIN) examples/component-choice build -a $(ARCH) --confirm
```

Comment:
NIT (that I'm a little bit more passionate about); I think `$(ZARF_BIN) package create examples/{EXAMPLE} build/ -a $(ARCH) --confirm)` is a lot more clear. Shortcutting away the 'package create' doesn't really seem necessary in a Makefile.